File: flight_insipiration_search/v1/get/iOS SDK/FlightInspirationSearch.swift
import WAConvo

var waconvo: WAConvo = WAConvo(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

waconvo.shopping.flightDestinations.get(params: ["origin": "MAD"],
    onCompletion: { result in
        switch result {
        case let .success(response):
            print(response.data)
        case let .failure(error):
            fatalError(error.localizedDescription)
        }
    })

File: flight_insipiration_search/v1/get/Python SDK/flight_inspiration_search.py
# Install the Python library from https://pypi.org/project/waconvo/# Install the Python library from https://pypi.org/project/waconvo
from waconvo import Client, ResponseError

waconvo = Client(
    client_id='YOUR_WACONVO_API_KEY',
    client_secret='YOUR_WACONVO_API_SECRET'
)

try:
    '''
    Find cheapest destinations from Madrid
    '''
    response = waconvo.shopping.flight_destinations.get(origin='MAD')
    print(response.data)
except ResponseError as error:
    raise error
